README.md

```diff
@@ -19,38 +19,50 @@ involved! We welcome all contributions.
 
 The ServerExe directory contains the server-side components of
 SeamlessRDP: the SeamlessRDP shell and window hooks. It uses a
-autotools-based build system.
+autotools-based build system. It is designed to support both 32 and
+64bit applications and therefore building is a bit complicated.
 
 
-## Building from a source archive (seamlessrdp-1.0.tar.gz)
+## Build instructions (Linux)
 
-    cd seamlessrdp-1.0
-    ./configure
-    make
+You need to have both mingw32 and mingw64 installed to be able to
```

Comment:
For Fedora, you want mingw32-gcc and mingw64-gcc installed. You then run `mingw32-configure` or `mingw64-configure` instead of `./configure`, then `make` as usual. You don't have to bother with knowing the host triplets for 32- and 64-bit environments.

On Ubuntu, installing mingw-w64 will get the cross-compiler for both 32 and 64-bit applications set up properly. The host triplet on Ubuntu is `i686-w64-mingw32` and `x86_64-w64-mingw32`, and the regular build instructions are OK.

Reply:
Maybe it's better if we tell our users the specific packages they need (on a few select distros) rather than the generalized info that might not be helpful enough for some.

Reply:
>For Fedora, you want mingw32-gcc and mingw64-gcc installed. You then run mingw32-configure or mingw64-configure instead of ./configure, then make as usual. You don't have to bother with knowing the host triplets for 32- and 64-bit environments.

It works perfectly fine to run ./configure --host=<triplet> on fedora so with that in mind, these instruction does not differs from whenever you are on ubuntu or fedora. Only the way how to get the two compilers installed, which we do not mention and care about.

Reply:
Alright. I still think there's value in describing how to set up the cross-compiling environment, but we can leave that for a wiki page or something.